repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -345,34 +433,42 @@ class Interpreter(prompt0: Ref[String],
           )
         )
     )
-  } yield res ++ hookImports
+  } yield (res._1 ++ hookImports, res._2)
 
-  def processExec(code: String): Res[Imports] = for {
-    (processedBlocks, hookImports) <- preprocessScript(ImportHook.Source.File(wd/"<console>"), code)
-    res <- processCorrectScript(
-      processedBlocks,
-      eval.sess.frames.head.imports ++ hookImports,
-      Seq(Name("$sess")),
-      Name("cmd" + eval.getCurrentLine),
-      { (processed, wrapperIndex, indexedWrapperName) =>
-        evaluateLine(
-          processed,
-          printer,
-          s"Main$wrapperIndex.scala",
-          indexedWrapperName
-        )
-      }
-    )
-  } yield res ++ hookImports
+
+
+  def processExec(code: String): Res[Imports] = {
+    init()
```

Comment:
For this code should I put `init` inside `_  <- Res.Success(init())` in for comprehension that will just add one line, instead of changing whole function?